cmd/agent/install_script.sh

```diff
@@ -661,13 +684,49 @@ else
       no_start=true
     fi
   fi
-  if [ "$site" ]; then
-    printf "\033[34m\n* Setting SITE in the $nice_flavor configuration: $config_file\n\033[0m\n"
-    $sudo_cmd sh -c "sed -i 's/# site:.*/site: $site/' $config_file"
-  fi
-  if [ -n "$DD_URL" ]; then
-    printf "\033[34m\n* Setting DD_URL in the $nice_flavor configuration: $config_file\n\033[0m\n"
-    $sudo_cmd sh -c "sed -i 's|# dd_url:.*|dd_url: $DD_URL|' $config_file"
+
+  if [ -z "$fips_mode" ]; then
+    if [ "$site" ]; then
+      printf "\033[34m\n* Setting SITE in the $nice_flavor configuration: $config_file\n\033[0m\n"
+      $sudo_cmd sh -c "sed -i 's/# site:.*/site: $site/' $config_file"
+    fi
+    if [ -n "$DD_URL" ]; then
+      printf "\033[34m\n* Setting DD_URL in the $nice_flavor configuration: $config_file\n\033[0m\n"
+      $sudo_cmd sh -c "sed -i 's|# dd_url:.*|dd_url: $DD_URL|' $config_file"
+    fi
+  else
```

Comment:
Let's print a message here about configuring the agent for the FIPS proxy, something like the messages in the `if` clause of this condition.